src/pure/stack_set.rs

```diff
@@ -1277,6 +1309,26 @@ pub mod tests {
 
         assert!(matches!(res, Err(Error::NoScreens)));
     }
+
+
+    #[test_case(true,1,1; "forward")]
```

Comment:
Clippy probably wont pick up on this but can you add spaces between the macro arguments here please?